src/components/file_upload.vue

```diff
@@ -210,6 +210,10 @@ export default class FileUpload extends Vue {
   position: relative;
   text-align: center;
   width: 100%;
+
+  * {
+    pointer-events: none;
```

Comment:
The "Upload from your computer" button is no longer clickable.

Reply:
Oops